packages/shared/src/components/post/PostItemCard.tsx

```diff
@@ -43,7 +43,14 @@ export default function PostItemCard({
   const article = post?.sharedPost ?? post;
 
   return (
-    <article>
+    <article
+      className={classNames(
```

Comment:
and this doesn't work for when it is clickable?
Wondering if we can simply uniform the two basically?